src/amber.cr

```diff
@@ -5,8 +5,7 @@ require "colorize"
 require "secure_random"
 require "kilt"
 require "kilt/slang"
-require "./amber/dsl/*"
-require "./amber/support/*"
+require "./amber/*"
```

Comment:
You don't need this line, there is already `require "./amber/**"` on the next line